network/p2p/inspector/validation/utils.go

```diff
@@ -1,14 +1,12 @@
 package validation
 
-import "github.com/onflow/flow-go/network/channels"
+type duplicateStrTracker map[string]struct{}
```

Comment:
Please add a happy path test for it.

Reply:
https://github.com/onflow/flow-go/pull/4574/commits/bb87e90d2d932bb3053fe5aebdcce9e693491874